### PR TITLE
Update readme with images, friendly instructions and conda env instructions

### PR DESCRIPTION
**Pull request by tomsherborne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
-# nlp-in-10-lines
-Repository for PyCons 2016 workshop Natural Language Processing in 10 Lines of Code.
+### PyCon UK 2016 
+![NLP in 10 Lines of Code](http://i.imgur.com/TuusTAm.png)
```
Comment (andrazhribernik) [inline README.md]: I would host this image on Github and add relative link to image hosted in this repo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+# Natural Language Processing in 10 Lines of Code
+##### from Andraz Hribernik, Matic Horvat, Aeneas Wiener, Tom Sherborne
+
+At [Cytora](www.cytora.com) we use NLP to extract and analyse plain text to build our structured information product.
```
Comment (andrazhribernik) [inline README.md]: Link does not work. Maybe you need to change `www.cytora.com` to `http://www.cytora.com`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+
+This is the repo for [our talk at PyCon UK](http://2016.pyconuk.org/workshops/natural-language-processing-in-10-lines-of-code/).
```
Comment (andrazhribernik) [inline README.md]: I would change `our talk at PyCon UK` to `our workshop at PyCon UK`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,40 @@
-# nlp-in-10-lines
-Repository for PyCons 2016 workshop Natural Language Processing in 10 Lines of Code.
+### PyCon UK 2016 
+![NLP in 10 Lines of Code](http://i.imgur.com/TuusTAm.png)
+# Natural Language Processing in 10 Lines of Code
+##### from Andraz Hribernik, Matic Horvat, Aeneas Wiener, Tom Sherborne
```
Comment (andrazhribernik) [inline README.md:4]: I am not sure we really need to have authors specified (usually contributors on Github are shown with commits), but if @aeneaswiener and @matichorvat like it, I do not mind.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,40 @@
...
+
+At [Cytora](www.cytora.com) we use NLP to extract and analyse plain text to build our structured information product.
+
+This is the repo for [our talk at PyCon UK](http://2016.pyconuk.org/workshops/natural-language-processing-in-10-lines-of-code/).
+In this repository you will find the step by step tutorial from the workshop on some basic Natural Language Processing tasks using [spaCy](http://spacy.io/),
+a powerful (and super fast) NLP model.
```
Comment (andrazhribernik) [inline README.md]: Do you think `NLP model` is better then `NLP library` or `NLP tool`?

File: README.md [hunk 6]
```diff
@@ -1,2 +1,40 @@
...
+
+To install the spaCy model you need to run:
+
+    sputnik --name spacy --repository-url http://index.spacy.io install en==1.1.0
+
+To run jupyter notebook run:
+
+    jupyter notebook
+
+Tutorial notebook could be found in `tutorial.ipynb`.
```
Comment (andrazhribernik) [inline README.md]: Maybe `tutorial.ipynb` could be changed to link [tutorial.ipynb](tutorial.ipynb)

File: README.md (hunk 5 above)
Comment (tomsherborne) [reply README.md]: Nah, I like library more. I'll change it

File: README.md (hunk 4 above)
Comment (tomsherborne) [reply README.md:4]: In my mind this was mostly an SEO thing to help your name come up when people search for spaCy but I can also take this out 😄

**Comment by tomsherborne** [comment]
@andrazhribernik updated and is now all functioning as it should

File: README.md (hunk 6 above)
Comment (tomsherborne) [reply README.md]: Should I link to this when this is in a different branch? I can link to the master version when its ready

**Pull request merged**